docs/guide/writing/support-team.rst

```diff
@@ -48,23 +48,31 @@ Often this means that you have a huge backlog of tasks to juggle, and you cannot
+
+Simplify communication
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When you are writing documentation full-time, you have little to no time to answer calls or respond to tickets. You are relying on your team to let you know about any issues that resurface enough to create a doc page or FAQ. However, they are busy too. Anyone with exposure to support work will know that it is a tough job that requires helping as many customers as you can in a single day while maintaining the highest satisfaction rating possible. The support team needs to be able to send you their issues quickly and efficiently.
 
-It is always important to remember that when you work as a separate role within a team, it does not in fact make you apart from the team. Even though you write, and they support, you are still one team who are essential to each other. The better the documentation is, the less redundancies there are in tickets and calls coming in. This means that support will get less tickets, yes, but those tickets will contain challenging issues that really require their time and skills. This is a rewarding, symbiotic relationship. When they have meetings on issues, you attend; when you have doc issues, you communicate those with them. You can’t be a team unless you understand each other and the unique issues each of you face. 
+There are several ways to do this:
 
-Make communication simple
-~~~~~~~~~~~~~~~~~~~~~~~~~
+* Create a dedicated email account for doc tasks where they can send issues.
+* Create a Slack command that assigns you the task automatically.
+* Use a team Trello board.
 
-When you are writing the documentation full-time, you have no time to answer calls or respond to tickets. You are relying on your team to let you know which issues are coming in often enough to create a doc page or FAQ question. However, they are busy too. Anyone with exposure to support work will know that it is tough job that requires helping as many customers as you can in a single day while maintaining the highest satisfaction rating possible. So, they need to be able to send you their issues quickly and efficiently. There are several ways to do this, creating a dedicated email account for doc tasks where they can send issues, creating a Slack command that assigns you the task automatically, having a team Trello board, and so on. Find the solution that works best for your team. And don’t despair – if one solution does not work, move on to the next.
+Find a solution that works best for your team, and don’t despair. If one solution does not work, move on to the next.
 
 Collaborate
-~~~~~~~~~~~
+~~~~~~~~~~~~
```

Comment:
Is this correct? Looks to be one character too long.

Reply:
Thank you for reviewing! 

I've checked [reST specification]( http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections). It says: 

> An underline/overline is a single repeated punctuation character that begins in column 1 and forms a line extending **at least as far as the right edge of the title text**

So I think, it should be fine. 

But I decided to trim underlines because it looks more tidy. :)